docker/policies/latest_tag.rego

```diff
@@ -17,6 +17,17 @@ __rego_input__ := {
 	"selector": [{"type": "dockerfile"}],
 }
 
+get_aliases[aliases] {
+	from_cmd := docker.from[_]
+	count(from_cmd.Value) == 3
+	aliases := from_cmd.Value[2]
+}
+
+is_alias(img) = allow {
```

Comment:
Since this is not allowing or denying anything, I would remove the `allow`, which will have the same result just more readable

Reply:
fixed